src/pyfmi/fmi_algorithm_drivers.py

```diff
@@ -159,6 +159,18 @@ class AssimuloFMIAlgOptions(OptionBase):
             example is filter = "*der" , stor all variables ending with
             'der'. Can also be a list.
             Default: None
+            
+        synchronize_simulation --
+            If set, the simulation will be syncronized to real-time or a
+            scaled real-time, if possible. The available options are:
+                True: Simulation is syncronized to real-time

```

Comment:
I think you accidentally dropped the letter h from the word syncronized

Reply:
Right, fixed.